ibis/backends/tests/test_generic.py

```diff
@@ -1442,7 +1442,24 @@ def test_hash_consistent(backend, alltypes):
     h1 = alltypes.string_col.hash().execute(limit=10)
     h2 = alltypes.string_col.hash().execute(limit=10)
     backend.assert_series_equal(h1, h2)
-    assert h1.dtype in ("i8", "uint64")  # polars likes returning uint64 for this
+
+
+@pytest.mark.notimpl(["dask", "pandas", "postgres", "risingwave", "flink", "exasol"])
+@pytest.mark.notyet(
+    [
+        "sqlite",
+        "datafusion",
+        "druid",  # not sure what's going on here
+        "mysql",  # CHECKSUM TABLE but not column
+        "trino",  # checksum returns varbinary
+    ]
+)
+def test_hash_is_int64(alltypes):
+    # some backend's hash() return a uint64, not a int64.
+    # Check that these backends successfully cast the result to int64
+    # so it is consistent with the other backends.
+    res = alltypes.string_col.hash().cast(str).cast("int64").execute(limit=10)
+    assert res.dtype == "i8"
```

Comment:
I'd just combine this test with the above, since it's effectively one more line in the existing test (and all the marks are the same).

```suggestion
    # Check that dtype is consistently i8
    assert h1.dtype == "i8"
```